src/samples/coap/iotivity-test-client.c

```diff
@@ -107,24 +107,24 @@ found_resource_print(struct sol_oic_client *cli, struct sol_oic_resource *res, v
     }
 
```

Comment:
+1 for using `printf`. Use `SOL_WRN` for output makes no sense, it should be used to log problems.